### ISSUE TITLE
Please, add the Schedule definition export into default index

### ISSUE THREAD

**Issue by yadickson** [roccivic/cron-converter#23]
Hello,

This is wonderful application.

When I use the Scheduler definition, I need to import from dist folder like this:

```
import { getSchedule, stringToArray } from "cron-converter";
import { Schedule } from "cron-converter/dist/mjs/schedule";
```

Please,

Is possible to add the Scheduler export definition into the default index, like this:

`import { getSchedule, stringToArray, Schedule } from "cron-converter";`

Thanks a lot

**Comment by roccivic** [comment]
@yadickson I just published `2.0.1`, it addresses your request. Can you please check that it works as you expect? Thanks again for your contribution.

**Comment by yadickson** [comment]
thanks a lot
